File: week6/problemset6/mario.py
from cs50 import get_int

# get height
n = get_int("Height: ")

# ask again if height is > 8 or < 1
while n < 1 or n > 8:
    n = get_int("Height: ")

# print hashes/spaces for pyramid
for i in range(1, n+1):
    # output progressing spaces
    for j in range(0, n-i):
        print(" ", end="")
    # output hashes
    for j in range(0, i):
        print("#", end="")
    print()

File: week1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompt how tall - set limits for between 1 and 8 as input
    int n;
    do
    {
        n = get_int("Height? ");
    }
    while (n < 1 || n > 8);
    // build loop //build subloops for hashes and spaces
    for (int i = 1; i <= n; i++)
    {
        // output progressing spaces
        for (int j = 0; j < n - i; j++)
        {
            printf(" ");
        }
        // output j hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
